### DISCUSSION TITLE
refresh page

### DISCUSSION THREAD

**Discussion by hoangde2811** [flet-dev/flet#1212] [Q&A]
Dear developer!!
When I refresh the page, the data is not reloaded, but the operation state remains. I have attached a video to show more clearly. Please help me to be able to run. 
Sincerely thank you.
(https://github.com/flet-dev/flet/files/11027156/video_display_error.zip)

**Comment by gbrandt** [comment]
Try to break down the issue to the smallest sample code and place the code in this ticket.  Rather than a video we need to be able to reproduce it.

(Plus I did not even download the video...a zip file that can contain almost anything is not a great idea to file a bug)

**Reply by hoangde2811** [reply]
Dear developer!
I sent the error back with the link attached below. Please take a look to understand my problem. Sincerely thank you
https://drive.google.com/file/d/1O1SQq0dl06RcEvAAhT0I32k1SHWbgKds/view

**Comment by hoangde2811** [comment]
Dear developer!
I sent the error back with the link attached below. Please take a look to understand my problem. Sincerely thank you
https://drive.google.com/file/d/1O1SQq0dl06RcEvAAhT0I32k1SHWbgKds/view

**Comment by FeodorFitsner** [comment]
I agree with @gbrandt - tapping into videos or zips shared here doesn't look safe. Please paste a code here or share a link to a repository.

**Reply by hoangde2811** [reply]
Hi
I have resend the link about the problem of not reloading the page in the browser. Help me. Sincerely thank you.
https://drive.google.com/file/d/1O1SQq0dl06RcEvAAhT0I32k1SHWbgKds/view

**Reply by hoangde2811** [reply]
Dear developer!!
I have attached my source (https://drive.google.com/drive/folders/1lva7Hym0lheAiS2vvIgDe2zBcsHsHN0y?usp=sharing). When I enter my username and password then I click the reload button on the browser bar and it doesn't reload but it keeps what I entered earlier. Please help me.
Sincerely thank you.

**Comment by gbrandt** [comment]
The video shows the issue but unless we can see the code we can’t tell what is causing the issue. Please try to reproduce the problem with the shortest code possible and post your source here.

**Reply by hoangde2811** [reply]
Dear developer!!
I have attached my source (https://drive.google.com/drive/folders/1lva7Hym0lheAiS2vvIgDe2zBcsHsHN0y?usp=sharing). When I enter my username and password then I click the reload button on the browser bar and it doesn't reload but it keeps what I entered earlier. Please help me. 
Sincerely thank you.
